testbed/tests/widgets/properties.py

```diff
@@ -21,14 +21,14 @@ async def test_enabled(widget, probe):
 
     # Disable the widget
     widget.enabled = False
-    await probe.redraw()
+    await probe.redraw(message="Widget should be disabled")
```

Comment:
The `message=` portion isn't really needed; there isn't any ambiguity in arguments, so there's no need to be explicit.